jrnl/upgrade.py

```diff
@@ -144,6 +156,7 @@ def upgrade_jrnl_if_necessary(config_path):
         j.write()
 
     print("\nUpgrading config...", file=sys.stderr)
+
```

Comment:
Please be careful committing random changing to different parts of files that what you're working in.